users/users.md

```diff
@@ -189,9 +189,9 @@ Response
 ```
 ---
 
-## Disable user account
+## Update user account
```

Comment:
I suggest removing "account" from this file. We have get user, get user list apis. I think we should make it consistent and have same convention, e.g. create user, update user, etc.

Reply:
updated